### PR TITLE
Implement EmitCFILsda function in the objwriter

### PR DESCRIPTION
**Pull request by janvorli**
This change implements the EmitCFILsda function necessary for Unix exception
handling.

### CODE REVIEW

**Comment by janvorli** [comment]
@kyulee1, @AndyAyersMS can you take a look please?

File: lib/ObjWriter/.nuget/runtime.json [hunk 1]
```diff
@@ -2,17 +2,17 @@
   "runtimes": {
     "win7-x64": {
       "Microsoft.DotNet.ObjectWriter": {
-        "toolchain.win7-x64.Microsoft.DotNet.ObjectWriter": "1.0.12-prerelease-00001"
+        "toolchain.win7-x64.Microsoft.DotNet.ObjectWriter": "1.0.13-prerelease-00001"
```
Comment (kyulee1) [inline lib/ObjWriter/.nuget/runtime.json]: There is the corresponding file `toolchain.win7-x64.Microsoft.DotNet.ObjectWriter.nuspec` which also needs to be updated. Same as other platforms.

File: lib/ObjWriter/objwriter.cpp [hunk 2]
```diff
@@ -539,6 +539,21 @@ extern "C" void EmitCFIEnd(ObjectWriter *OW, int Offset) {
   OW->FrameOpened = false;
 }
 
+extern "C" void EmitCFILsda(ObjectWriter *OW,  const char *LsdaBlobSymbolName)
+{
```
Comment (kyulee1) [inline lib/ObjWriter/objwriter.cpp]: This should be in the above line per convention.

File: lib/ObjWriter/objwriter.cpp [hunk 3]
```diff
@@ -539,6 +539,21 @@ extern "C" void EmitCFIEnd(ObjectWriter *OW, int Offset) {
...
+  assert(OW->FrameOpened && "frame should be opened before CFILsda");
+  auto *AsmPrinter = &OW->getAsmPrinter();
+  auto &OST = static_cast<MCObjectStreamer &>(*AsmPrinter->OutStreamer);
+  MCContext &OutContext = OST.getContext();
+
+  // Create symbol reference
+  MCSymbol *T = OutContext.getOrCreateSymbol(LsdaBlobSymbolName);
+  MCAssembler &MCAsm = OST.getAssembler();
+  MCAsm.registerSymbol(*T);
+  OST.EmitCFILsda(T, llvm::dwarf::Constants::DW_EH_PE_pcrel | llvm::dwarf::Constants::DW_EH_PE_sdata4);
```
Comment (kyulee1) [inline lib/ObjWriter/objwriter.cpp]: If you have clang in your window box, I normally run `python utils\ccformat.py --untidy` to format this automatically.
```
  OST.EmitCFILsda(T, llvm::dwarf::Constants::DW_EH_PE_pcrel |
                         llvm::dwarf::Constants::DW_EH_PE_sdata4);
```

**Comment by janvorli** [comment]
I've reflected the feedback and augmented the commit. @kyulee1 can you please take another look?

**Comment by kyulee1** [comment]
LGTM

**Comment by kyulee1** [comment]
@janvorli Currently there is no automation for packaging. I will let you know once I'm done so that you can change CoreRT yourself including version update.

**Comment by janvorli** [comment]
@kyulee1 Great, thanks a lot!

**Pull request merged**